.evergreen.yml

```diff
@@ -15,15 +15,15 @@ mongo_tools_variables:
       mongod_args: ""
       mongod_port: 33333
     ssl: &mongod_ssl_startup_args
-      mongod_args: "--sslMode requireSSL --sslCAFile db/testdata/ca.pem --sslPEMKeyFile db/testdata/server.pem"
+      mongod_args: "--sslMode requireSSL --sslCAFile ./db/testdata/ca-ia.pem --sslPEMKeyFile ./db/testdata/test-server.pem"
```

Comment:
was it not working without the `./`?

Reply:
it seems working before. But I changed it to a new certificate here.